packages/shared/src/components/layout/MainLayoutHeader.tsx

```diff
@@ -86,10 +86,63 @@ function MainLayoutHeader({
       extra: JSON.stringify({ notifications_number: unreadCount }),
     });
   };
-  const { referralToken } = useReferralCampaign({
+  const { referralToken, isReady } = useReferralCampaign({
```

Comment:
This also works when user is not eliged for it? (just making sure)

Reply:
Yes, I tried incognito and it still renders the UI.

Reply:
Also validated from the docs.

![Screenshot 2023-10-24 at 12 57 22 AM](https://github.com/dailydotdev/apps/assets/13744167/f5cfc9a8-9c8d-4d37-b787-fc30082b8aa6)